libs/vscode/nx-cloud-view/src/lib/nx-cloud-service/nx-cloud-service.ts

```diff
@@ -468,7 +466,7 @@ export class NxCloudService extends StateBaseService<InternalState> {
   private async loadAndSetIsPrivateCloud() {
     const nxWorkspaceConfig = (await getNxWorkspace()).workspace;
     const cloudRunnerUrl =
-      nxWorkspaceConfig.tasksRunnerOptions?.default?.runner === '@nrwl/nx-cloud'
```

Comment:
here we need to check for both `@nrwl/nxcloud` and `nx-cloud` runners because this feature should work regardless of your nx version.